run-custom-mao.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash -xe
+
+export SCRIPTS_DIR=$(pwd)
+export CUSTOM_MAO_BUILD=quay.io/sdasu/machine-api-operator:sdasu-fix
+
+kubectl get secrets --namespace openshift-machine-api | grep mariadb
+
+# Stop CVO, MAO, machine-api-controller and Metal3 deployments
```

Comment:
There is also a stop_mao.sh script here in this repository that takes the approach previously described in the documentation. It just tells the CVO to stop managing the MAO, rather than stopping the CVO entirely. Does that make a difference?

Reply:
That works too.